bridge_ui/src/hooks/useCheckIfWormholeWrapped.ts

```diff
@@ -101,7 +102,7 @@ function useCheckIfWormholeWrapped(nft?: boolean) {
     (async () => {
       if (isEVMChain(sourceChain) && sourceAsset) {
         // use JsonRpcProvider if both source chain and target chain are evm chains
-        const provider = evmChainId === sourceChain ? evmProvider : getEvmJsonRpcProvider(sourceChain)
+        const provider = evmChainId === getEvmChainId(sourceChain) ? evmProvider : getEvmJsonRpcProvider(sourceChain)
```

Comment:
We should convert the wormhole chain id to the EVM chain id before comparing. I checked the entire codebase; this is the only place with an issue.